src/components/Inputs/ButtonSelectGroup/ButtonSelectGroup.js

```diff
@@ -52,16 +52,19 @@ export const ButtonSelectGroup = ({
   fullWidth = true,
   ...rest
 }) => {
-  const initialSelected = currentValue || initialValue || undefined 
+  const initialSelected =
+    currentValue !== undefined ? currentValue : initialValue
```

Comment:
initialValue defaults to undefined so the `initialValue || undefined` step didn't do anything. also, it would have been undefined anyway if it didn't default to undefined... haha